src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySink.java

```diff
@@ -254,12 +257,18 @@ private void validateConfiguredSchema(Schema schema, FailureCollector collector)
       return;
     }
 
-    Table table = BigQueryUtil.getBigQueryTable(config.getProject(), config.getDataset(), config.getTable(),
-                                                config.getServiceAccountFilePath(), collector);
-    if (table != null) {
-      // if table already exists, validate schema against underlying bigquery table
+    String tableName = config.getTable();
+    Table table = BigQueryUtil.getBigQueryTable(config.getProject(), config.getDataset(), tableName,
+      config.getServiceAccountFilePath(), collector);
 
-      validateSchema(table, schema, config.allowSchemaRelaxation, collector);
+    if (tableName != null) {
```

Comment:
Please make sure this is using correct variable. This will cause null pointer exception if table does not exist:
https://github.com/data-integrations/google-cloud/pull/384

Reply:
fixed